packages/s3-bridge/src/Resources/config/s3-bridge.yaml

```diff
@@ -0,0 +1,26 @@
+parameters:
```

Comment:
too many services.yaml files... it isn't easy to locate the proper one when you're searching files... 

I can rename it if you insist on the name

Reply:
One more does not change a thing for me so I would vote to rename it as it will be unified. We have parameters for example in services.yaml of the framework, so I would not mind having it in one file.

Reply:
renamed in 75199e6a6